### PR TITLE
Update windows base image versions for integ tests

### PR DESCRIPTION
**Pull request by jterry75**
Signed-off-by: Justin Terry <jlterry@amazon.com>

### Summary

This updates the windows base images for the integ test so we actually use the latest ones.

### Implementation details

Pulls them by ltsc tag where applicable.

### Testing

`.\run-integ-tests.ps1`

New tests cover the changes: 

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/run-integ-tests.ps1 [hunk 1]
```diff
@@ -12,40 +12,39 @@
...
 # Prepare windows base image
 $dockerImages = Invoke-Expression "docker images"
 if (-Not ($dockerImages -like "*$BaseImageName*")) {
   Invoke-Expression "docker pull $BaseImageName"
 }
 Invoke-Expression "docker tag $BaseImageName amazon-ecs-ftest-windows-base:make"
 
+$imgDigest = Invoke-Expression "docker image inspect --format `"{{.RepoDigests}}`" $BaseImageName"
+# The template returns an array so its [$BaseImageDigest]. Remove the '[' and ']'
+$imgDigest.SubString(1, $imgDigest.Length - 2)
```
Comment (rawahars) [inline scripts/run-integ-tests.ps1:42]: Should this be `$imgDigest = $imgDigest.SubString(1, $imgDigest.Length - 2)`?

Comment (jterry75) [reply scripts/run-integ-tests.ps1:42]: How the heck. That is saved in my local file. I'm not sure how git did this. Will update you are totally right

**Review by rawahars — APPROVED** [review]
LGTM

**Review by vsiddharth — DISMISSED** [review]

**Comment by jterry75** [comment]
This should have been against dev. Moving

**Pull request closed without merging**